modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -206,6 +206,8 @@ def __copy__(self):
             call_queue=self.call_queue,
         )
 
+    _iloc = ray.put(lambda df, row_labels, col_labels: df.iloc[row_labels, col_labels])
```

Comment:
```suggestion
    _iloc = ray.put(PandasDataframePartition._iloc)
```
to not duplicate the implementation